kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/WorkflowRunner.kt

```diff
@@ -95,23 +98,23 @@ interface WorkflowRunner<out OutputT> {
      * It's probably more convenient to use [FragmentActivity.setContentWorkflow]
      * rather than calling this method directly.
      */
-    @UseExperimental(ObsoleteCoroutinesApi::class)
-    fun <InputT, OutputT : Any> of(
+    @UseExperimental(ObsoleteCoroutinesApi::class, ExperimentalCoroutinesApi::class)
+    fun <InputT : Any, OutputT : Any> of(
```

Comment:
Had to add this constraint because the `Flowable.asFlow()` extension is only defined for non-nullable types. And it's just more correct anyway.